specs/definitions/signed_docs/docs/comment_moderation_action.cue

```diff
@@ -4,6 +4,8 @@ package signed_docs
 
 docs: #DocumentDefinitions & {
 	"Comment Moderation Action": {
+		draft: true
```

Comment:
If your going to add `draft` then i think it should also be expressed in the documentation `draft: false` no change.  `draft: true` put a disclaimer in the document that the document type is still being defined and subject to large changes.

Don't use the word "finalized" because that implies immutability and we don't guarantee immutability of any document format at this stage.  So this flag is just a signal to you that this document is just a placeholder and its basic behavior is still undefined.

Reply:
updated